packages/server-wallet/src/wallet/store.ts

```diff
@@ -158,7 +159,7 @@ export class Store {
       type: 'NotifyApp' as 'NotifyApp',
       notice: {
         method: 'MessageQueued' as 'MessageQueued',
-        params: {sender, recipient, data},
+        params: serializeMessage(data, recipient, sender),
```

Comment:
`serializeMessage` again. This one is in the store, so the `channelId` should be readily available.